libraries/node-core-library/src/Sort.ts

```diff
@@ -231,4 +231,79 @@ export class Sort {
       set.add(item);
     }
   }
+
+  /**
+   * Sort the keys given in an object
+   *
+   * @example
+   *
+   * ```ts
+   * console.log(Sort.sortKeys({ c: 3, b: 2, a: 1 })); // { a: 1, b: 2, c: 3}
+   * ```
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public static sortKeys<T extends Partial<Record<string, any>> | Array<any>>(
+    object: T,
+    { deep, compare }: { deep?: boolean; compare?: (x: string, y: string) => number } = {
+      deep: false,
+      compare: Sort.compareByValue
+    }
+  ): T {
+    function isPlainObject(obj: unknown): boolean {
+      return obj !== null && typeof obj === 'object';
+    }
+    if (!isPlainObject(object) && !Array.isArray(object)) {
+      throw new TypeError(`Expected object or array`);
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Most of these `no-explicit-any` warnings can be eliminated by using `unknown` instead of `any`, since this code mostly does not care about the structure of those values.